spec/pushmi_pullyu/aip_spec.rb

```diff
@@ -1,69 +1,75 @@
 require 'spec_helper'
 
-# Please note that much of the nitty-gritty is tested in the specs for creator and downloader
-
 RSpec.describe PushmiPullyu::AIP do
-  let(:workdir) { 'tmp/aip_spec' }
-  let(:options) do
-    { workdir: workdir,
-      fedora: { url: 'http://www.example.com:8983/fedora/rest',
-                base_path: '/dev',
-                user: 'fedoraAdmin',
-                password: 'fedoraAdmin' },
-      solr: { url: 'http://www.example.com:8983/solr/development' } }
-  end
   let(:noid) { '9p2909328' }
-  let(:mock_download_data) { "spec/fixtures/aip_download/#{noid}" }
-  let(:aip_file) { "#{workdir}/#{noid}.tar" }
+  let(:workdir) { 'tmp/aip_spec' }
+  let(:aip_folder) { "#{workdir}/#{noid}" }
+  let(:aip_file) { "#{aip_folder}.tar" }
 
   before do
-    allow(PushmiPullyu.logger).to receive(:info)
-    allow(PushmiPullyu.logger).to receive(:debug)
-    allow(PushmiPullyu).to receive(:options) { options }
-    FileUtils.mkdir_p(workdir)
-    FileUtils.cp_r(mock_download_data, workdir)
+    allow(PushmiPullyu).to receive(:options) { { workdir: workdir } }
   end
 
-  after do
-    FileUtils.rm_rf(workdir)
-    FileUtils.rm_rf(aip_file)
-  end
+  describe '.create' do
```

Comment:
Changed most of these. The actual work of these are being done in the creator/downloader classes, which we are already testing. Here we just want to see if, we are calling instancing those classes and running them, and that we are returning the filename/directory as expected.